app/utils/cookies.js

```diff
@@ -163,12 +228,22 @@ function getSessionVariable (req, key) {
   return session && session[key]
 }
 
+function getSessionChargeState (req, key) {
+  return getSessionVariable(req, key).data
```

Comment:
the call sites for getSessionChargeState have handling code for if the returned value is `undefined`, this may be the case for any existing charges in flight prior to the merge of this PR